src/api/feed-discovery/server.js

```diff
@@ -0,0 +1,11 @@
+const { Satellite } = require('@senecacdot/satellite');
+
+const feedDiscoverRouter = require('./src/router');
+
+const service = new Satellite();
```

Comment:
You might want to split this file in two to make testing easier, see my code in https://github.com/Seneca-CDOT/telescope/pull/1893/files where I have `src/index.js` and `src/server.js` to make getting at the service's `app` instance easier.